src/docs/patterns/empty-states.mdx

```diff
@@ -1,64 +1,89 @@
+-   Success Screens
 
-## Loading Data
+Empty states should **NOT** be used for:
 
-<ImageGrid
-    images={[Image1, Image2]}
-    caption={'When data is loading, use placeholder shapes for where the data will appear.'}
-/>
-<DemoCard repository={'loading-states'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Floading-states%2Floading-states.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/loading-states/index.tsx&initialpath=loading-states'} ionic reactNative />
+-   [Loading indicators](/patterns/loading)
+-   Welcome Screens
 
-When loading data for the first time, you do not want to show an empty screen while you are fetching data. Instead, you should show placeholders that mimic (using simple shapes) the data that will eventually be loaded. This will give the impression that your application is faster and gives users an indication of what kind of data is being loaded.
+<Divider />
 
-The placeholder should match as closely as possible the real structure of the data. For example, if you are loading a list where each item has an icon, two lines of text, and an action button, then your placeholder should have a circle, two rectangles for the text, and another rectangle for the button. You should show as many placeholder elements as necessary to fill the area in the application that is loading data.
+# Anatomy
 
-There are a number of libraries available to help you create this placeholder content, such as:
+Empty states should include a large icon or graphic followed a brief headline/message. If necessary, additional explanatory text may follow.
 
--   [@netbasal/ngx-content-loader](https://www.npmjs.com/package/@netbasal/ngx-content-loader) (Angular, Ionic)
--   [ion-skeleton-text](https://ionicframework.com/docs/api/skeleton-text) (Ionic)
--   [placeholder-loading](https://www.npmjs.com/package/placeholder-loading) (React, Angular, Ionic)
--   [react-content-loader](https://www.npmjs.com/package/react-content-loader) (React)
--   [rn-placeholder](https://www.npmjs.com/package/rn-placeholder) (React Native)
+The text on the screen should clearly indicate why there is no data to display, and what (if anything) can be done to make data available. The text should be minimal, but enough to convey the necessary message.
 
-### Refreshing Data
+If the data is empty but could be populated by a user action (such as adding a device to an empty device list), there should be a call-to-action button that will allow the user to add data.
 
-When refreshing data, there are several approaches you can take. If you are showing real-time data (either via polling or a push mechanism from the server), there is no need to show large loading indicators. You may wish to show a small indicator or at least mention when the data was last refreshed so users can identify if the data is stale.
+<ImageGrid images={[Image1, Image2]} />
```

Comment:
Replace those images too with the latest designs from figma... the screenshots feel outdated